### DISCUSSION TITLE
Use of API REST in the system CRON

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#701] [General]
Hi,
I did a planned mailing. The mailing appears in "Email Management/Mailings", 

<img width="1696" alt="ScheduleActions" src="https://github.com/user-attachments/assets/cb435eda-af87-448a-9077-42867ff57bcd" />

but not in the tab "Schedule Actions" as you can see.
<img width="494" alt="Mailings" src="https://github.com/user-attachments/assets/5ff1dbd1-67bd-4060-96c7-b5557f044482" />

Mails only appear to be queued at the time they are scheduled to be sent, and as they remain there for less than a minute, it's hard to see why.
Wouldn't it be better to make them appear as soon as they are scheduled?

**Comment by liedekef** [comment]
A mailing is calculated the moment it is send out (not individual mails), so as to make sure the correct people are included (imagine you send it to a group one month in the future and the group changes).
But I'll try to include some info about planned mailings too ...

**Reply by liedekef** [reply]
The change is now in dev: https://github.com/liedekef/events-made-easy/commit/1bb61037e59967c19757995de25a6926645149b8

**Reply by f14m07** [reply]
Thanks a lot Franky, I prefere now. 
Happy New Year in some days ;-)
